client/my-sites/stats/stats-subscribers-overview/index.tsx

```diff
@@ -1,57 +1,90 @@
 import { CountComparisonCard } from '@automattic/components';
+import { UseQueryResult } from '@tanstack/react-query';
 import { translate } from 'i18n-calypso';
 import React from 'react';
+import useSubscribersQuery from 'calypso/my-sites/stats/hooks/use-subscribers-query';
 
-function SubscribersOverviewCardStats() {
-	const overviewCardStats = [
-		{
-			heading: translate( 'Today' ),
-			count: 1410,
-		},
-		{
-			heading: translate( '30 days ago' ),
-			count: 1080,
-			previousCount: 1080,
-		},
-		{
-			heading: translate( '60 days ago' ),
-			count: 1000,
-			previousCount: 1080,
-		},
-		{
-			heading: translate( '90 days ago' ),
-			count: 970,
-			previousCount: 1080,
-		},
-	];
-	return overviewCardStats;
+interface SubscribersData {
+	period: string;
+	subscribers: number;
+	subscribers_change: number;
 }
 
-function SubscribersOverviewCards() {
-	const overviewCardStats = SubscribersOverviewCardStats();
+interface SubscribersDataResult {
+	data: SubscribersData[];
+	unit: string;
+	date: string;
+}
 
-	return (
-		<div className="overview-cards-list">
-			{ overviewCardStats.map( ( overviewCardStat ) => (
-				<CountComparisonCard
-					key={ overviewCardStat.heading }
-					heading={ overviewCardStat.heading }
-					count={ overviewCardStat.count }
-					previousCount={ overviewCardStat.previousCount }
-					showValueTooltip
-					icon={ false }
-				/>
-			) ) }
-		</div>
-	);
+interface SubscribersOverviewProps {
+	siteId: number;
+}
+
+function extractCountsAtIndexes( subscribersData: SubscribersData[], indexes: number[] ): number[] {
+	return indexes.map( ( index ) => subscribersData[ index ]?.subscribers || 0 );
+}
+
+// all comparisons are being compared to todays's count to show growth up to today
+function SubscribersOverviewCardStats( subscribersData: number[] ) {
+	const daysToDisplay = [ 0, 30, 60, 90 ];
+	const overviewCardStats: {
+		heading: string;
+		count: number;
+		previousCount?: number;
+	}[] = [];
+
+	daysToDisplay.forEach( ( day, index ) => {
+		const count = subscribersData[ index ] || 0;
+
+		const cardStat = {
+			heading: ( index === 0
+				? translate( 'Today' )
+				: translate( '%d days ago', { args: day } ) ) as string,
+			count: count,
```

Comment:
`previousCount` is missing at the moment.

Reply:
correct. After chatting with @jsnmoon yesterday afternoon I decided to pull out the previous count change indicators and move my doubts to a P2 which I will be posting today and will hopefully explain everything for now 😄

Reply:
Anna is correct — our current design specifications lead to an undesirable/misleading interface for a site with growing subscriber counts. Definitely merits further discussion in a P2 🙂